### DISCUSSION TITLE
I hope that a strong physics library will be added

### DISCUSSION THREAD

**Discussion by WOLFxxxxxx** [raysan5/raylib#4609] [General]
https://github.com/amerkoleci/joltc/tree/e63fd48d8f5579641910e1ce0c97e83215f21c59

Jolt c

**Comment by raysan5** [comment]
no plans.

**Comment by orcmid** [comment]
@WOLFxxxxxx

Jolt is serious C++.  The derivation of a C API to it (JoltC) is seriously undocumented and it also requires C++ compilations to build.  The C Language header introduces an extensive number of typedefs, many of which conflict in usage (not names) with the ones defined for raymath and used in raylib (vectors and matrices in particular).

Rather than dreaming of this, consider that it is a nightmare in the context of raylib.

@raysan5 says it best.
